tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -26,16 +26,20 @@
 
 INSTALL_LOCAL_REQUIREMENTS_CMD = ["python", "-m", "pip", "install", "-r", "requirements.txt"]
 INSTALL_CONNECTOR_PACKAGE_CMD = ["python", "-m", "pip", "install", "."]
+INSTALL_POETRY_PACKAGE_CMD = ["python", "-m", "pip", "install", "poetry"]
+POETRY_INSTALL_DEPENDENCIES_CMD = ["poetry", "install"]
+
 DEFAULT_PYTHON_EXCLUDE = [".venv"]
+POETRY_EXCLUDE = ["__pycache__"] + DEFAULT_PYTHON_EXCLUDE
 CI_CREDENTIALS_SOURCE_PATH = "tools/ci_credentials"
 CI_CONNECTOR_OPS_SOURCE_PATH = "tools/ci_connector_ops"
```

Comment:
nit: what do you think about moving these constant directly to the environment function they're used in. I think it my help for readability and they are not reused.

Reply:
Generally in favor! With the exception of the paths.

Love getting the path in constants pattern set up early so we dont have to play find where we hardcoded the path later

Reply:
Sounds good to me.